frontend/src/pages/modelRegistry/screens/utils.ts

```diff
@@ -176,3 +176,21 @@ export const isValidHttpUrl = (value: string): boolean => {
     return false;
   }
 };
+
+export const filterCustomProperties = (
+  customProperties: ModelRegistryCustomProperties,
+  keys: string[],
+): ModelRegistryCustomProperties => {
+  const filteredCustomProperties: ModelRegistryCustomProperties = {};
+  Object.keys(customProperties).forEach((key) => {
+    if (!keys.includes(key)) {
+      filteredCustomProperties[key] = customProperties[key];
+    }
+  });
+  return filteredCustomProperties;
+};
+
+export const isPipelineRunExist = (
+  customProperties: ModelRegistryCustomProperties,
+  keys: string[],
+): boolean => keys.every((key) => key in customProperties);
```

Comment:
If we're moving to `getRegisteredFromPipelineProps`, we could have it return `null` if all the required properties aren't present and then we wouldn't need `isPipelineRunExist`.